cmd/minikube/cmd/status.go

```diff
@@ -35,15 +43,35 @@ var statusCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
 		defer api.Close()
-		s, err := cluster.GetHostStatus(api)
+		ms, err := cluster.GetHostStatus(api)
+		if err != nil {
+			glog.Errorln("Error getting machine status:", err)
+			os.Exit(1)
+		}
+		ls := "N/A"
+		if ms == state.Running.String() {
+			ls, err = cluster.GetLocalkubeStatus(api)
+		}
 		if err != nil {
 			glog.Errorln("Error getting machine status:", err)
 			os.Exit(1)
 		}
-		fmt.Fprintln(os.Stdout, s)
+		status := Status{ms, ls}
+
+		tmpl, err := template.New("status").Parse(statusFormat)
+		if err != nil {
+			glog.Errorln("Error creating status template:", err)
+			os.Exit(1)
+		}
+		err = tmpl.Execute(os.Stdout, status)
+		if err != nil {
+			glog.Errorln("Error executing status template:", err)
+			os.Exit(1)
+		}
 	},
 }
 
 func init() {
+	statusCmd.Flags().StringVar(&statusFormat, "format", constants.DefaultStatusFormat, "Go template format string for the status output.  Go templates format can be found here: https://golang.org/pkg/text/template/\nFor the list accessible variables for the template, see the struct values here:https://godoc.org/k8s.io/minikube/cmd/minikube/cmd#Status")
```

Comment:
This might be easier to read with a multiline string literal:

`my string
hello
yo
`